modules/selenium_module.py

```diff
@@ -269,5 +269,8 @@ def capture_host(cli_parsed, http_object, driver, ua=None):
         with open(http_object.source_path, 'w') as f:
             f.write('Cannot render webpage')
         http_object.headers = {'Cannot Render Web Page': 'n/a'}
+    except IOError:
+        print("[*] ERROR: URL too long, surpasses max file length.")
```

Comment:
This issue also occurred when the `source` folder did not exist in the results directory